docs/core/howto/endpoints.rst

```diff
@@ -152,6 +152,30 @@ Keep in mind that you may need to wrap this up for your particular application,
 For example, that little snippet is slightly oversimplified: at the time ``connectedNow`` is run, the bot hasn't authenticated or joined the channel yet, so its message will be refused.
 A real-life IRC bot would need to have its own method for waiting until the connection is fully ready for chat before chatting.
 
+Reporting an Initial Failure
+----------------------------
+
+Often times, a failure of the very first connection attempt is special. It may indicate a problem that won't go away by just trying harder. The service may be configured with the wrong hostname, or the user may not have an internet connection at all (perhaps they forgot to turn on their wifi adapter).
```

Comment:
Semantic linefeed is the preferred formatting for docs.  Put a linefeed after `.`